src/funcapp.jl

```diff
@@ -1,74 +1,304 @@
-	
+using FastGaussQuadrature
+using Gadfly
+using Base.Test
+using ApproxFun
+using ApproXD
+using Plots
 
```

Comment:
you need to put those things **inside** the module. you clearly didn't try to `include("run.jl")`. because that doesn't work!